backend/src/routes/rentalAgency.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+const express  = require('express');
+const router   = express.Router();
+
+const RentalAgencyController = require('../controllers/rentalAgency');
+
+
+router.get('/', RentalAgencyController.list); // List all rental agencies
+router.post('/', RentalAgencyController.create); // Add a new rental agency
```

Comment:
Unusable.

Reply:
Changed to only support search for nearby agencies